client/my-sites/stats/stats-page-loader/index.tsx

```diff
@@ -80,7 +85,7 @@ export default function StatsPageLoader( { children }: AsyncLoadProps ) {
 		canCurrentUser( state, siteId, 'view_stats' )
 	);
 	const jetpackModuleActive = useSelector( ( state ) =>
-		isJetpackModuleActive( state as object, siteId, 'stats' )
+		isJetpackModuleActive( state as object, siteId, 'stats', true )
```

Comment:
I wonder under what circumstances do we use the fallback option? Asking because we have a couple of other instances that are using the selector without the last param.

It seems like it fallbacks to Jetpack options, but it'll be great to be sure.

cc/ @Automattic/red